core/src/client.rs

```diff
@@ -56,6 +59,7 @@ const HEADER_WAREHOUSE: &str = "X-DATABEND-WAREHOUSE";
 const HEADER_STAGE_NAME: &str = "X-DATABEND-STAGE-NAME";
 const HEADER_ROUTE_HINT: &str = "X-DATABEND-ROUTE-HINT";
 const TXN_STATE_ACTIVE: &str = "Active";
+const HEADER_SQL: &str = "SQL";
```

Comment:
X-DATABEND-SQL?

Reply:
lgtm，since we prefer user to use bendsql instead of curl

Reply:
done